.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "OpenFeature .NET SDK",
+  "image": "mcr.microsoft.com/devcontainers/dotnet:9.0-bookworm",
+  "features": {
+    "ghcr.io/devcontainers/features/dotnet:latest": {
+      "version": "9.0",
+      "additionalVersions": "8.0"
+    },
+    "ghcr.io/devcontainers/features/github-cli:latest": {},
+    "ghcr.io/devcontainers/features/docker-in-docker": {}
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "EditorConfig.EditorConfig",
+        "GitHub.copilot",
+        "GitHub.copilot-chat",
+        "GitHub.vscode-github-actions",
+        "GitHub.vscode-pull-request-github",
+        "ms-dotnettools.csharp",
+        "esbenp.prettier-vscode",
+        "redhat.vscode-yaml",
+        "cucumberopen.cucumber-official",
+        "ms-dotnettools.csdevkit"
+      ]
+    }
+  },
+  "remoteUser": "vscode",
+  "hostRequirements": {
+    "memory": "8gb"
```

Comment:
Are you sure this much member is necessary? It seems a bit excessive.

Reply:
I just want to give the 2 core CPU which requires the 8gb in code spaces. Also for the samples it is better to be a bit "bigger" machine.